modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_info.jsp

```diff
@@ -149,8 +149,12 @@ List<Folder> mountFolders = DLAppServiceUtil.getMountFolders(scopeGroupId, DLFol
 
 <c:if test="<%= windowState.equals(WindowState.MAXIMIZED) %>">
 	<aui:script>
-		Liferay.Util.focusFormField(
```

Comment:
Hi @ambrinchaudhary, @boton,

The purpose of this code is to perform an `autoFocus` on the search box. Do you think it would be a good idea to issue this as a new requirement for clay's management toolbar? They could add an `autoFocus` attribute so that we can get rid of this code in the future.

Reply:
Hi @4lejandrito, yes, it seems that it could be a common pattern and thus it could be included in the managment toolbar. I suggest you to open an issue on [Clay](https://github.com/liferay/clay/issues) so they can think if this really fits for them.

Reply:
I'm not sure but looks like a good idea :D

Reply:
https://github.com/liferay/clay/issues/3895